diff --git a/utils/bazel/llvm-project-overlay/llvm/config.bzl b/utils/bazel/llvm-project-overlay/llvm/config.bzl
index 2309175d04a9..45916b410825 100644
--- a/utils/bazel/llvm-project-overlay/llvm/config.bzl
+++ b/utils/bazel/llvm-project-overlay/llvm/config.bzl
@@ -43,7 +43,6 @@ posix_defines = [
     "HAVE_REGISTER_FRAME=1",
     "HAVE_SETENV_R=1",
     "HAVE_STRERROR_R=1",
-    "HAVE_SYSEXITS_H=1",
     "HAVE_SYS_IOCTL_H=1",
     "HAVE_UNISTD_H=1",
 ]
